data/json/items/corpses.json

```diff
@@ -0,0 +1,156 @@
+    "category": "other",
+    "material": "hflesh",
+    "price": 0,
+    "volume": "7500 ml",
+    "flags": [ "TRADER_AVOID" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_generic_male",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "A dead body of a middle-aged man.",
+    "looks_like": "corpse",
+    "material": "hflesh"
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_generic_female",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "A dead body of a young woman.",
+    "looks_like": "corpse",
+    "volume": "40000 ml",
+    "weight": "52000 g",
+    "material": "hflesh"
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_generic_boy",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "A dead body of a little boy.",
+    "looks_like": "corpse",
+    "material": "hflesh"
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_generic_girl",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "A dead body of a little girl.",
+    "looks_like": "corpse",
+    "material": "hflesh"
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_child_calm",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "Facial expression of this child is surprisingly calm.  It seems that he died instantly and most probably didn't even know that he is dying.  You could even say he's a lucky to die this way in this crazy world.",
+    "looks_like": "corpse",
+    "volume": "30000 ml",
+    "weight": "40750 g",
+    "material": "hflesh"
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_child_gunned",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "You can't even say for sure if it's a boy or a girl because the body of this poor little creature has been literally riddled by bullets.  It's beyond your understanding what kind of a monster could have done this to a child.",
+    "looks_like": "corpse",
+    "volume": "30000 ml",
+    "weight": "40750 g",
+    "material": "hflesh"
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_bloody",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "A dead body lying in the puddle of its own blood.",
+    "looks_like": "corpse",
+    "material": "hflesh"
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_painful",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "It's hard to tell how exactly did this person die, but judging from his twisted body and frightful grimace, it was a really painful death.",
+    "looks_like": "corpse",
+    "material": "hflesh"
+  },
+  {
+    "type": "GENERIC",
+    "id": "corpse_scorched",
+    "copy-from": "corpse",
+    "name": "corpse",
+    "description": "An awful mass of charred flesh.  You're not even sure if it's a man or a woman.  Judging by the stain of vaporized blood next to the body, he probably was shot prior to the burning.",
```

Comment:
"Not sure if it's a man or a woman" maybe change "he probably" to "they probably"?